v2/pkg/operator/local_stored_collector.go

```diff
@@ -40,9 +42,11 @@ type LocalStorageCollector struct {
 func (o *LocalStorageCollector) OperatorImageCollector(ctx context.Context) ([]v1alpha3.CopyImageSchema, error) {
 
 	var (
-		allImages []v1alpha3.CopyImageSchema
-		label     string
-		dir       string
+		allImages      []v1alpha3.CopyImageSchema
+		label          string
+		dir            string
+		imageReference string
+		imageName      string
```

Comment:
Is it probably safer to reset these 2 to empty at the beginning of the for loop?